qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AdministrationPage.java

```diff
@@ -16,7 +16,8 @@
 public class AdministrationPage extends Page {
 
 	private static final String URL = "/admin/index.htm";
-	private static final By MANAGE_USERS = By.cssSelector("#content a[href='/openmrs/admin/users/users.list']");
+	private static final By MANAGE_USERS= By.cssSelector("#content a[href='/openmrs/admin/users/users.list']");
```

Comment:
```suggestion
	private static final By MANAGE_USERS = By.cssSelector("#content a[href='/openmrs/admin/users/users.list']");
```

Is `MANAGE_USERS` a link or a button?